.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
+python: 3.5
```

Comment:
Travis won't have Python 3.5 present in the build environment without this line, that's why I have it added here.